shared/common-adapters/button.desktop.js

```diff
@@ -13,6 +13,9 @@ class Button extends Component {
     let backgroundStyle = {}
     let labelStyle = {}
     let progressColor = globalColors.white
+    let rippleStyle: {rippleColor: string} = {}
+    // Set all button ripples to be more prominent
+    rippleStyle = {rippleColor: 'rgba(0, 0, 0, 0.3)'}
```

Comment:
Style nit: I think it would make sense to combine these lines for now:

```
// Set all button ripples to be more prominent
const rippleStyle: {rippleColor: string} = {rippleColor: 'rgba(0, 0, 0, 0.3)'}
```